src/components/window-shade.sass

```diff
@@ -30,5 +31,43 @@
   padding-bottom: 28px;
 
 .windowShadeContentHide
-  height: 0px
+  height: 3px
   overflow: hidden
+  border-top: #878706 solid 3px
+
+.content
+  font-family: Arial, sans-serif
+  font-size: 16pt
+  font-weight: regular
+  line-height: 24pt
+  margin-bottom: 12pt
+
+.theory-and-background
```

Comment:
I'd prefer camelCase all the time as that's easier to use in JS/JSX (`css.theoryAndBackground` instead of `css["theory-and-background"]`).

Reply:
Agreed. Looks much better.